package.json

```diff
@@ -102,50 +100,50 @@
     "babel-plugin-inline-react-svg": "andyrichardson/babel-plugin-inline-react-svg#refs",
     "babel-plugin-styled-components": "^1.12.0",
     "bestzip": "^2.2.0",
-    "clean-webpack-plugin": "^3.0.0",
+    "clean-webpack-plugin": "^4.0.0-alpha.0",
     "codemirror": "^5.61.0",
-    "codemirror-graphql": "^0.12.1",
+    "codemirror-graphql": "^1.0.1",
     "concurrently": "^5.3.0",
-    "copy-webpack-plugin": "^6.1.0",
-    "csp-html-webpack-plugin": "^4.0.0",
-    "css-loader": "^3.6.0",
+    "copy-webpack-plugin": "^8.1.1",
+    "csp-html-webpack-plugin": "^5.1.0",
+    "css-loader": "^5.2.4",
     "d3-scale": "^3.3.0",
-    "dotenv": "^8.2.0",
+    "dotenv": "^8.5.0",
     "enzyme": "^3.11.0",
     "enzyme-to-json": "^3.6.2",
     "eslint": "^7.25.0",
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-react": "^7.23.2",
     "fast-json-stable-stringify": "^2.1.0",
-    "graphql": "^14.7.0",
-    "graphql-tag": "^2.11.0",
-    "html-webpack-plugin": "^4.4.1",
+    "graphql": "^15.5.0",
```

Comment:
Would be worth doing a quick pass here to double check there are no breaking changes (mainly when we do the introspection query to the client) - https://github.com/graphql/graphql-js/releases/tag/v15.0.0

Reply:
I had a look and everything looks good from my perspective, it will be great to have second eyes